src/coreclr/gc/env/common.h

```diff
@@ -22,6 +22,7 @@
 #include <stdarg.h>
 #include <memory.h>
 #include <limits.h>
+#include <math.h>
```

Comment:
This probably isn't needed anymore (fabsf was the original reason), though I imagine we'll take some dependence on it at some point

Reply:
n/m I assume this is for `log`